content/developer/reference/addons/testing.rst

```diff
@@ -523,15 +525,163 @@ Tips
 Integration Testing
 ===================
 
-Testing Python code and JS code separately is very useful, but it does not prove
-that the web client and the server work together.  In order to do that, we can
-write another kind of test: tours.  A tour is a mini scenario of some interesting
-business flow.  It explains a sequence of steps that should be followed.  The
-test runner will then create a phantom_js browser, point it to the proper url
-and simulate the click and inputs, according to the scenario.
+Testing Python code and JS code separately is very useful, but it does not prove that the web client
+and the server work together.  In order to do that, we can write another kind of test: tours.  A
+tour is a mini scenario of some interesting business flow.  It explains a sequence of steps that
+should be followed.  The test runner will then create a phantom_js browser, point it to the proper
+url and simulate the click and inputs, according to the scenario.
+
+Writing a test tour
+-------------------
+
+Structure
+~~~~~~~~~
+
+To write a test tour for `your_module`, start with creating the required files:
+
+.. code-block:: text
+
+    your_module
+    ├── ...
+    ├── static
+    |   └── src
+    |       └── js
+    |           └── tours
+    |               └── your_tour.js
+    ├── tests
+    |   ├── __init__.py
+    |   └── test_calling_the_tour.py
+    ├── views
+    |   └── your_module_views.xml
+    └── __manifest__.py
+
+You can then:
+
+- add your :file:`your_tour.js` as an asset in :file:`your_module_views.xml`.
+- update :file:`__manifest__.py` to load :file:`your_module_views.xml`.
+- update :file:`__init__.py` in the folder :file:`tests` to import :file:`test_calling_the_tour`.
+
+.. seealso::
+   - :doc:`../javascript/javascript_reference`, section :ref:`js_reference/adding_assets`
+   - :ref:`testing/python`
+
+Javascript
+~~~~~~~~~~
+
+1. Setup your tour by registering it:
```

Comment:
```suggestion
#. Setup your tour by registering it:
```
This allows automatically updating the numbering should someone add/remove an item.